### PR TITLE
813 sub task antegen consolidate duplicate documentation

### LINKED ISSUE #813: [sub-task ANTeGen] Consolidate duplicate documentation
The test level table, example prompts, fixture registration instructions, and review caveats 
are duplicated across three locations:
- `docs/agent_network_test_generator.md` (detailed reference — source of truth)
- `docs/user_guide.md` (should be brief intro + link)
- `registries/agent_network_test_generator.hocon` (LLM needs inline copy)
 
Tasks:
- [ ] Trim `user_guide.md` "Automatic Test Generation" section to a short paragraph + link to `agent_network_test_generator.md`
- [ ] Trim duplicated "Register the fixture" section in `user_guide.md` to a link
- [ ] Add sync comment in HOCON above test_level definitions: `# Canonical descriptions in docs/agent_network_test_generator.md — keep in sync.`
 
Reviewer: @deepsaia

### PR DESCRIPTION
**Pull request by vince-leaf**
This PR is the fix that @deepsaia has suggested in the 1st release of ANTeGen.

 Summary of the 3 changes:

- Trim user_guide.md "Automatic Test Generation" replace with a short paragraph + link to agent_network_test_generator.md
Trim user_guide.md "Register the fixture"  replace with a brief sentence + link
Add sync comment in HOCON (registries/agent_network_test_generator.hocon) — # Canonical descriptions in docs/agent_network_test_generator.md — keep in sync.

### CODE REVIEW

File: docs/user_guide.md [hunk 1]
```diff
@@ -1733,31 +1733,13 @@ or manually.
...
-
-You can control how many scenarios are generated with the `test_level` parameter:
-
-```text
-Generate test cases for basic/coffee_finder_advanced with minimum coverage
-Generate test cases for basic/music_nerd_pro with max coverage
-```
-
-| Level     | Scenarios | Description                                                        |
-|-----------|-----------|--------------------------------------------------------------------|
```
Comment (vince-leaf) [inline docs/user_guide.md:1752]: replaced with a short paragraph

File: docs/user_guide.md [hunk 2]
```diff
@@ -1733,31 +1733,13 @@ or manually.
...
-| `minimum` | 2--3      | One core happy-path and one critical edge case.                    |
-| `normal`  | 5--7      | Happy paths, edge cases, and at least one error/boundary scenario. |
-| `max`     | 10--15    | Exhaustive coverage of all capabilities and edge cases.            |
-
-The generator saves fixtures under `tests/fixtures/<agent_name>/`. Review the generated
-files before committing — you may need to adjust `gist` criteria, `keywords`, `value`
-checks, or `sly_data` values. See
-[Agent Network Test Generator](agent_network_test_generator.md) for full details.
+For test levels, example prompts, and review tips, see the full
+[Agent Network Test Generator](agent_network_test_generator.md) documentation.
```
Comment (vince-leaf) [inline docs/user_guide.md:1742]: replaced with a short paragraph

File: docs/user_guide.md [hunk 3]
```diff
@@ -1772,28 +1754,8 @@ For existing examples, look at the fixtures under `tests/fixtures/` in this repo
...
-    ),
-    skip_on_empty=True,
-)
-```
-
-Each test method is tagged with `@pytest.mark` markers that mirror the folder grouping
-(e.g. `integration_basic`, `integration_basic_coffee_finder_advanced`). If you are adding
-fixtures for a new agent network that does not have an existing test method, create a new
-method following the same pattern.
+See [Running a Generated Test Fixture — Option B](agent_network_test_generator.md#option-b-register-in-the-integration-test-suite)
```
Comment (vince-leaf) [inline docs/user_guide.md:1757]: replaced with a short paragraph

File: registries/agent_network_test_generator.hocon [hunk 4]
```diff
@@ -109,6 +109,7 @@ Each scenario you return must include:
 - `description`: what the scenario tests.
 - `interactions`: a list of the EXACT user messages for each turn, with sly_data and expected behaviour. For multi-turn scenarios, this list MUST contain multiple entries (one per turn). For example, a 3-turn scenario MUST have 3 interaction entries.
 
+# Canonical descriptions in docs/agent_network_test_generator.md — keep in sync.
```
Comment (vince-leaf) [inline registries/agent_network_test_generator.hocon:112]: Added the comment to keep in sync

**Review by deepsaia — APPROVED** [review]
LGTM.
